explanation/intro-to/high-availability.md

```diff
@@ -16,40 +16,37 @@ A definition of High Availability Clusters [from Wikipedia:](https://en.wikipedi
 
-### Example
+### Example of HA cluster quorum
 
 ![2-node HA cluster|578x674,75%](https://assets.ubuntu.com/v1/14896401-HA_intro.png)
 
 ## Fencing
 
-Fencing protects your data from being corrupted, and your application from becoming unavailable, due to unintended concurrent access by rogue nodes.
+Fencing protects your data from being corrupted and prevents your application from becoming unavailable due to unintended concurrent access by rogue nodes. If a node is unresponsive doesn’t mean it has stopped accessing your data. The only way to be absolutely sure that your data is safe is to employ fencing, which establishes that the unresponsive node is completely offline before the data is accessed by another node.
 
-Just because a node is unresponsive doesn’t mean it has stopped accessing your data. The only way to be 100% sure that your data is safe, is to use fencing to ensure that the node is truly offline before allowing the data to be accessed from another node.
+Also, if a clustered service cannot be stopped, a cluster can use fencing to force the whole node offline and making it safe to start the service elsewhere. The most popular example of fencing is cutting a host’s power.
 
-Fencing also has a role to play in the event that a clustered service cannot be stopped. In this case, the cluster uses fencing to force the whole node offline, thereby making it safe to start the service
-elsewhere. The most popular example of fencing is cutting a host’s power.
+### Key benefits of fencing
 
-Key Benefits:
+- An active counter-measure taken by a functioning host to isolate a misbehaving (usually dead) host from shared data.
 
-- Active counter-measure taken by a functioning host to isolate a misbehaving (usually dead) host from shared data.
-
-- **MOST CRITICAL** part of a cluster utilising SAN or other shared storage technology (*Ubuntu HA Clusters can only be supported if the fencing mechanism is configured*).
+- A mechanism that is the **most critical** part of a cluster utilising SAN or other shared storage technology (*Ubuntu HA Clusters can only be supported if the fencing mechanism is configured*).
 
 - Required by OCFS2, GFS2, cLVMd (before Ubuntu 20.04), lvmlockd (from 20.04 and beyond).
 
-## Linux High Availability Projects
+## Linux High Availability projects
 
-There are many upstream high availability related projects that are included in Ubuntu Linux. This section will describe the most important ones.
+This section will only describe the most important upstream high availability related projects that are included in Ubuntu Linux. There are many other HA projects that are available to Ubuntu.
```

Comment:
This line is a bit longer now, but it's not really clearer. Could we revert this change?